src/features/lazy-load.feature

```diff
@@ -0,0 +1,24 @@
+@llcssbg @setup
```

Comment:
`@llcssbg` tag is used for all `ll-css-bg-` feature tests. Let's add another one dedicated for lazy-load.feature and use`@llimages`